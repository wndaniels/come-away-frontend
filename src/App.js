import Paths from "./Paths/Paths.js";
import "./App.css";

function App() {
  return <Paths />;
}

export default App;
