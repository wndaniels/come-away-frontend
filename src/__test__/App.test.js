import "../matchMedia.mock";
import React from "react";
import { render } from "@testing-library/react";
import App from "../App";

it("renders without crashingt", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
