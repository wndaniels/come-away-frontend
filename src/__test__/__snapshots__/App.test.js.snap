// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashingt 1`] = `
<DocumentFragment>
  <nav
    class="NavBar navbar navbar-expand-sm"
  >
    <div
      class="container-fluid"
    >
      <a
        class="navbar-brand"
        href="/"
      >
        COME/AWAY
      </a>
      <ul
        class="navbar-nav ml-auto"
      >
        <li
          class="nav-item mr-4"
        >
          <a
            class="nav-link"
            href="/login"
          >
            Login
          </a>
        </li>
        <li
          class="nav-item mr-4"
        >
          <a
            class="nav-link"
            href="/signup"
          >
            Sign Up
          </a>
        </li>
      </ul>
    </div>
  </nav>
  <div
    class="Home m-5"
  >
    <div
      class="container text-center"
    >
      <h1
        class="mb-4"
      >
        COME/AWAY
      </h1>
      <div>
        <p>
          Finally, a scheduler for expecting parents. No more trying to facilitate visiting while dealing with the stressors of your new child! Let us take care of that for you.
        </p>
        <p>
          <a
            class="btn btn-primary"
            href="/login"
          >
            Login
          </a>
          <a
            class="btn btn-primary ms-3"
            href="/signup"
          >
            Sign Up
          </a>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
